mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAO.java

```diff
@@ -197,7 +197,7 @@ private ByteBuffer toByteBuffer(InputStream stream) throws IOException {
             ids -> FluentFutureStream.of(
                 ids.stream()
                     .map(id -> retrieveRow(id, fetchType)
-                        .thenApply(resultSet ->
+                        .thenApply((ResultSet resultSet) ->
```

Comment:
I am wonderring if we should not let a command when we add optional typing (from a javac point of view) to fix a eclipse compilation error. Otherwise someone could remove the type one day and then a other eclipse developer then might encounter the same issue.